src/Component.php

```diff
@@ -97,6 +99,22 @@ public static function getName()
         return $fullName;
     }
 
+    public static function getPrefix()
+    {
+        return Str::startsWith(static::class, app('livewire')->getComponentNamespaces())
+            ? array_search(Arr::first(app('livewire')->getComponentNamespaces(), function ($namesapce) {
+                return str_starts_with(static::class, $namesapce);
+            }), app('livewire')->getComponentNamespaces())
+            : null;
```

Comment:
Looks like you're iterating the same array 3 times in a row. I'll send a PR to your branch with a simplified version of this algorithm :smiley:

Reply:
Thanks, merged 👌